studio/components/interfaces/App/FeaturePreviewModal.tsx

```diff
@@ -0,0 +1,97 @@
+import { useTelemetryProps } from 'common'
+import { useRouter } from 'next/router'
+import { useState } from 'react'
+import { Button, IconEye, IconEyeOff, Modal, ScrollArea, cn } from 'ui'
+
+import { LOCAL_STORAGE_KEYS } from 'lib/constants'
+import Telemetry from 'lib/telemetry'
+import { useAppStateSnapshot } from 'state/app-state'
+import { Markdown } from '../Markdown'
+import { useFeaturePreviewContext } from './FeaturePreviewContext'
+
+const FEATURE_PREVIEWS = [
```

Comment:
We should probably move this to a separate file, together with `LOCAL_STORAGE_KEYS`. We should make adding new feature previews as simple as possible.

Reply:
gotcha on this - i'll add this as a comment in the file itself for now